lib/link_state.h

```diff
@@ -169,6 +170,7 @@ struct ls_node {
 #define LS_ATTR_ADJ_SID6	0x04000000
 #define LS_ATTR_BCK_ADJ_SID6	0x08000000
 #define LS_ATTR_SRLG		0x10000000
+#define LS_ATTR_EXT_ADM_GRP 0x20000000
 
```

Comment:
Please align the define even if it is not mandatory

Reply:
It is because of `git clang-format`